insalan/user/models.py

```diff
@@ -147,9 +147,13 @@ def send_email_confirmation(user_object: User):
         send_mail(
             _("Confirmez votre courriel"),
             _("Confirmez votre adresse de courriel en cliquant sur ")
-            + "http://api."
+            + getenv("HTTP_PROTOCOL", "https")
```

Comment:
Nitpick: I think there is a global variable that already fetches this in `insalan.settings`